caffe2/python/onnx/tests/c2_ref_test.py

```diff
@@ -66,7 +66,7 @@ def test_relu_node_inplace(self):
             name="test",
             inputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])],
             outputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])])
-        c2_rep = c2.prepare(make_model(graph_def))
+        c2_rep = c2.prepare(make_model(graph_def), producer_name='caffe2-ref-test')
```

Comment:
You are now passing `producer_name` to `prepare`, not `make_model` :-)

Reply:
oops... ;-)